### ISSUE TITLE
Add link to homepage in the logo

### ISSUE THREAD

**Issue by manishprivet** [manishprivet/electify#16]
Generally in any website, the logo in the header should redirect to the homepage. We want to add that to the site

**Comment by wsameer** [comment]
Hey, i'd like to work on this issue.

**Comment by manishprivet** [comment]
Sure @wsameer Go ahead

**Comment by wsameer** [comment]
@manishprivet I need help with development set-up. 

stuck here
![image](https://user-images.githubusercontent.com/8468992/135755403-05f83391-6af0-4872-a0cc-8bbe9353fc0d.png)

**Comment by manishprivet** [comment]
@wsameer I don't think you need to set up the project for this issue.

You'll have to add the link here

https://github.com/manishprivet/electify/blob/40471b385e4ad3db7510383ebe18c11cd95f7655/components/Layout.tsx#L31

You can test it by running the project locally by running `npm run dev`

**Comment by wsameer** [comment]
> @wsameer I don't think you need to set up the project for this issue.
> 
> You'll have to add the link here
> 
> https://github.com/manishprivet/electify/blob/40471b385e4ad3db7510383ebe18c11cd95f7655/components/Layout.tsx#L31
> 
> You can test it by running the project locally by running `npm run dev`

![image](https://user-images.githubusercontent.com/8468992/135791668-4dacbf78-5896-4b57-b829-a1205dce066c.png)

I have added the code but unable to test it since the `npm run dev` command also gives error. @manishprivet

**Comment by wsameer** [comment]
@manishprivet is this still an active repo?

**Comment by manishprivet** [comment]
Yes @wsameer , Was busy with Uni work for past few days, apologies
